src/Xamarin.Android.NamingCustomAttributes/Android.App/ActivityAttribute.cs

```diff
@@ -31,6 +33,7 @@ public ActivityAttribute ()
 #if ANDROID_24
 		public bool                   DirectBootAware         {get; set;}
 #endif
+		public string                 EnableVrMode            {get; set;}
```

Comment:
The [VR docs state](https://developers.google.com/vr/reference/vr-manifest#vr_activity_attributes) that:

> The `android:enableVrMode` attribute enables Android VR Mode, a feature introduced in Android 7.0 (Nougat)

meaning it should be wrapped in `ANDROID_24`:

```csharp
#if ANDROID_24
		public string                 EnableVrMode            {get; set;}
#endif
```

This can in fact be merged with the `#if ANDROID_24` block containing `DirectBootAware`, above.

Reply:
Good catch.  Updated :)